code/modules/research/designs/autolathe_designs.dm

```diff
@@ -799,3 +799,11 @@
 	materials = list(MAT_METAL = 7500, MAT_GLASS = 100)
 	build_path = /obj/item/pet_carrier
 	category = list("initial", "Misc")
+
+/datum/design/miniature_power_cell
+	name = "Light Fixture Battery"
```

Comment:
also, item names should be `light fixture battery`

Reply:
Autolathe designs have always been capitalized.

Reply:
my mistake